src/sidebar/util/sentry.js

```diff
@@ -51,8 +51,28 @@ export function init(config) {
   Sentry.init({
     dsn: config.dsn,
     environment: config.environment,
-    // Do not log Fetch failures to avoid inundating with unhandled fetch exceptions
-    ignoreErrors: ['Fetch operation failed'],
+
+    // Ignore various errors due to circumstances outside of our control.
+    ignoreErrors: [
+      // Ignore transient network request failures. Some of these ought to be
+      // caught and handled better but for now we are suppressing them to
+      // improve the signal-to-noise ratio.
+      'Failed to fetch', // Chrome
+      'Fetch operation failed',
+      'NetworkError when attempting to fetch resource', // Firefox
+
+      // Ignore network request failures due to empty JSON bodies.
+      'JSON.parse: unexpected end of data', // Firefox
+      'Unexpected end of JSON input', // Opera Mobile
```

Comment:
I'm assuming that any time the client gets an incomplete JSON body, there was a problem somewhere between the client and the server.